playbooks/installation/load_balancing.adoc

```diff
@@ -0,0 +1,329 @@
+
+===== 1. Pre-requisites
+
+* Two Load Balancer VIPs Pre-created
+  ** External VIP
+    *** Configured for SSL Termination, using either:
+      **** A certificate signed by a Public Certificate Authority
+      **** A trusted certificate from your org's PKI
+    *** VIP must listen on port 8443, and proxy back to all master hosts on port 8443
+    *** For this example, we will give our External VIP a domain name: _paas.myorg.com_
+  ** Internal VIP (will be the same configuration as the Simple integration option above)
+    *** Configured for SSL Passthrough (see step 1 below)
+    *** VIP must listen on port 8443, and proxy back to all master hosts on port 8443
+    *** For this example, we will give our Internal VIP a domain name: _paas-internal.myorg.com_
+* Domain Name for each VIP registered in DNS
+  ** Domain name for _external_ VIP will become value of `openshift_master_cluster_public_hostname` in OpenShift Installer (i.e. paas.myorg.com)
+  ** Domain name for _internal_ VIP will become value of `openshift_master_cluster_hostname` in OpenShift Installer (i.e. paas-internal.myorg.com)
+* Master hosts created and prepped per link:https://docs.openshift.com/enterprise/3.1/install_config/install/prerequisites.html[Install Prerequisites]
+
+For the purposes of this example, we will use the hosts specified in the table below
+
+[options="header"]
+|===
+|Host Name |Infrastructure Component
+|*paas.myorg.com*
+|Pre-Configured *_External_* LB VIP with Public SSL Cert
+|*paas-internal.myorg.com*
+|Pre-Configured *_Internal_* LB VIP configured for SSL Passthrough
+|*master01.myorg.com*
+.3+.^|Master (clustered using native HA) and node
+|*master02.myorg.com*
+|*master03.myorg.com*
+
+|*etcd01.myorg.com*
+.3+.^|etcd Data Store
+|*etcd02.myorg.com*
+|*etcd03.myorg.com*
+
+|*node01.myorg.com*
+.2+.^|Node
+|*node02.myorg.com*
+|===
+
+////////////
+====== 1. Pre-create Master Certificates
+
+Because we need to use a cert created by OpenShift (signed by OpenShift's private CA), we need to pre-create certificates for OpenShift before actually running the installer. To do this, run this from the first master host, with the `atomic-openshift` package installed:
+
+[source]
+----
+# export service_names="kubernetes,kubernetes.default,kubernetes.default.svc,kubernetes.default.svc.cluster.local,openshift,openshift.default,openshift.default.svc,openshift.default.svc.cluster.local"
+# export master_hosts="master01.myorg.com,master02.myorg.com,master03.myorg.com,192.168.122.1,192.168.122.2,192.168.122.3"
+# oadm ca create-master-certs \
+  --hostnames=paas-internal.myorg.com,$master_hosts,172.30.0.1,$service_names \
+  --master=https://paas-internal.myorg.com:8443 \
+  --public-master=https://paas.myorg.com:8443 \
+  --cert-dir=/etc/origin/master
+----
+
+TIP: See link:https://docs.openshift.org/latest/install_config/upgrades.html#updating-master-certificates[Updating Master Certificates] for more details on filling out the `--hostnames` parameter.

```

Comment:
This section is commented out currently, and may be removed. I'm just going to leave it as is until we decide what to do with it